starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,60 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" href="css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600" rel="stylesheet">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+<div class="container">
+  <div class="headerimg">
+    <img src="images/apple-pie.jpg">
```

Comment:
something to look into: CSS background images.  This is what I usually use for situations like this.  Especially when text is supposed to go inside the image.